java/open-mobile-scene-package/README.md

```diff
@@ -0,0 +1,36 @@
+# Open Scene Map Package
```

Comment:
Should this be sentance case or are titles exempt?

Reply:
You're right...also the content of the title is non-sensical...long day...